### PR TITLE
Intent test habitevent deletion

### PR DESCRIPTION
**Pull request by j-vent**
Intent test for habit event deletion

## Description of Changes
Add a new habitevent to be deleted
Click on habitevent
Delete said habit event in habit event details page
Return back to habit list and check that there is one less habit in list

## Screenshots

image

## Checklist

- [X] Automated tests
- [x] Screenshots included (if necessary)
- [x] Code is well commented

Closes #ISSUE_ID

### CODE REVIEW

File: src/app/src/androidTest/java/com/cmput301f21t09/budgetprojectname/HabitEventTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+     *
+     * @throws Exception
+     */
+    @Test
+    public void start() throws Exception {
+        Activity activity = rule.getActivity();
+        System.out.println("printing!");
+    }
+
+    @Test
```
Comment (AtrazineC) [inline src/app/src/androidTest/java/com/cmput301f21t09/budgetprojectname/HabitEventTest.java:72]: Javadoc missing

**Review by AtrazineC — APPROVED** [review]
Looks great!

File: src/app/src/androidTest/java/com/cmput301f21t09/budgetprojectname/HabitEventTest.java (hunk 1 above)
Comment (j-vent) [reply src/app/src/androidTest/java/com/cmput301f21t09/budgetprojectname/HabitEventTest.java:75]: Added!

**Review by AtrazineC — APPROVED** [review]

File: src/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
             android:name=".ViewHabitEventActivity"
             android:exported="false" />
         <activity
-            android:name=".UserLoginActivity"
+            android:name=".ExampleActivity"
```
Comment (cheetiantey) [inline src/app/src/main/AndroidManifest.xml:32]: Shouldn't we use `UserLoginActivity` as default?

Comment (j-vent) [reply src/app/src/main/AndroidManifest.xml:32]: Just changed it for the purposes of testing this activity. Should I update it in this PR or just let people change it as they use it?

Comment (cheetiantey) [reply src/app/src/main/AndroidManifest.xml:32]: Yup, I think we can just let people change it as they see fit!

**Review by cheetiantey — APPROVED** [review]

**Pull request merged**